Obsidian/Utils/WADHashGenerator.cs

```diff
@@ -175,12 +174,13 @@ private static IEnumerable<string> ProcessBINLinkedFiles(List<string> linkedFile
         {
             List<string> strings = new List<string>();
 
-            for (int i = 0; i < linkedFiles.Count; i++)
+            foreach (string fetchedString in linkedFiles)
```

Comment:
change to 'for' as it's faster

Reply:
I actually was curious as to whether this is actually true, so I profiled the code and came to the following conclusion:
1. The loop only ever gets called with around 0-12 strings in ``linkedFiles``, which results in each finished loop taking not even 1/100 of a millisecond, which I consider to be negligible in time.
2. Even if I multiple the amount of elements per list by 1000, I wasn't able to notice any speed difference between ``foreach`` and ``for`` at all.

Considering these two points and the fact that using ``foreach`` results in cleaner code, I'd like to keep it this way.